static/js/protein/chart.ts

```diff
@@ -29,10 +31,63 @@ export function drawTissueScoreChart(
   id: string,
   data: { name: string; value: string }[]
 ): void {
-  // TODO: convert data to tmpData format.
-  const tmpData = [{ x1: 20, x2: 60, y1: 30, y2: 50 }];
+  const testData = data;
+  /*
+  Convert the tissue expression level to numerical values 
+  * @param val value of the data array 
+  */
+  function convertData(val) {
+    if (val == "ProteinExpressionNotDetected") {
+      return 0;
+    } else if (val == "ProteinExpressionLow") {
+      return 1;
+    } else if (val == "ProteinExpressionMedium") {
+      return 2;
+    } else if (val == "ProteinExpressionHigh") {
+      return 3;
+    }
+  }
+
+  interface newData{
+    name: string;
+    value: number;
+  }
+  const reformattedData = data.map((item) => {
+    var newObj = {} as newData
+    //var newObj: any = {};
+    newObj["name"] = item.name;
+    newObj["value"] = convertData(item.value);
+    return newObj;
+  });
+
+  const arr_name = reformattedData.map((x) => {
+    return x.name;
+  });
+
+  const height = 400 - MARGIN.top - MARGIN.bottom;
```

Comment:
400 and 460 should be defined constants

Reply:
Replaced the number values with constants.